docs/configurable-namespace.md

```diff
@@ -0,0 +1,89 @@
+# Configurable Namespace for C# Binding Generation
+
+This document explains how to configure the namespace for C# binding generation in the MAUI KMP project.
+
+## Overview
+
+The KSP processor now supports configurable namespace and prefix parameters for C# binding generation, instead of using hardcoded values.
+
+## Configuration Parameters
+
+### `mauikmp.csharpIOSBindingNamespace`
+- **Default**: `"Voize"`
+- **Description**: The namespace used for generated C# bindings
+- **Usage**: This will be the namespace that wraps all generated C# interfaces
+
+### `mauikmp.csharpIOSBindingPrefix`
+- **Default**: `"Shared"`
+- **Description**: The prefix used for generated C# class names
+- **Usage**: This prefix is prepended to all generated C# interface names
```

Comment:
this is the prefix which comes from the kotlin generated objectC headers, this can be configured in kotlin framework generation and must be added here to be the same.